
<!DOCTYPE html>
<html>
<meta charset="UTF-8">
<head>
  <title>Topic 10 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</title>

<style>
table {
  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #0099FF;
  color: white;
}
</style>

</head>
<body>
<h2>Topic 10 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</h2>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cite ad</th>
      <th>title</th>
      <th>authors</th>
      <th>publish year</th>
      <th>publish time</th>
      <th>dataset</th>
      <th>abstract mentions covid</th>
      <th>pmcid</th>
      <th>pubmed id</th>
      <th>doi</th>
      <th>cord uid</th>
      <th>topic weight</th>
      <th>Similarity scispacy</th>
      <th>Similarity specter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="jb447ndz";>1</th>
      <td>ponti_2018</td>
      <td>A method for RNA structure prediction shows evidence for structure in lncRNAs</td>
      <td>Riccardo Delli ponti; Alexandros Armaos; Stefanie Marti; Gian Gaetano Tartaglia</td>
      <td>2018</td>
      <td>2018-04-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/284869" target="_blank">10.1101/284869</a></td>
      <td>jb447ndz</td>
      <td>0.824928</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_10.html#scn7b4c6">Delli_Ponti_2018</a>, <a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="scn7b4c6";>2</th>
      <td>Delli_Ponti_2018</td>
      <td>A Method for RNA Structure Prediction Shows Evidence for Structure in lncRNAs</td>
      <td>Delli Ponti, Riccardo; Armaos, Alexandros; Marti, Stefanie; Tartaglia, Gian Gaetano</td>
      <td>2018</td>
      <td>2018-12-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6286970" target="_blank">PMC6286970</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30560136.0" target="_blank">30560136.0</a></td>
      <td><a href="https://doi.org/10.3389/fmolb.2018.00111" target="_blank">10.3389/fmolb.2018.00111</a></td>
      <td>scn7b4c6</td>
      <td>0.810154</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_10.html#jb447ndz">ponti_2018</a></td>
    </tr>
    <tr>
      <th id="6mb9w76f";>3</th>
      <td>Chen_2015</td>
      <td>Nuclear Magnetic Resonance-Assisted Prediction of Secondary Structure for RNA: Incorporation of Direction-Dependent Chemical Shift Constraints</td>
      <td>Chen, Jonathan\nL.; Bellaousov, Stanislav; Tubbs, Jason D.; Kennedy, Scott D.; Lopez, Michael J.; Mathews, David H.; Turner, Douglas H.</td>
      <td>2015</td>
      <td>2015-10-09</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4666457" target="_blank">PMC4666457</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26451676.0" target="_blank">26451676.0</a></td>
      <td><a href="https://doi.org/10.1021/acs.biochem.5b00833" target="_blank">10.1021/acs.biochem.5b00833</a></td>
      <td>6mb9w76f</td>
      <td>0.793799</td>
      <td><a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="l72x4wn3";>4</th>
      <td>Tagashira_2020</td>
      <td>PhyloFold: Precise and Swift Prediction of RNA Secondary Structures to Incorporate Phylogeny among Homologs</td>
      <td>Masaki Tagashira</td>
      <td>2020</td>
      <td>2020-03-06</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.05.975797" target="_blank">10.1101/2020.03.05.975797</a></td>
      <td>l72x4wn3</td>
      <td>0.778935</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_10.html#p1i7bx94">Zhang_2013</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="7pdh7alh";>5</th>
      <td>Hong_2013</td>
      <td>Viral IRES Prediction System - a Web Server for Prediction of the IRES Secondary Structure In Silico</td>
      <td>Hong, Jun-Jie; Wu, Tzong-Yuan; Chang, Tsair-Yuan; Chen, Chung-Yung</td>
      <td>2013</td>
      <td>2013-11-05</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818432" target="_blank">PMC3818432</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24223923.0" target="_blank">24223923.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0079288" target="_blank">10.1371/journal.pone.0079288</a></td>
      <td>7pdh7alh</td>
      <td>0.737691</td>
      <td><a href="Topic_10.html#wt0lvhfl">Wu_T_2009</a></td>
      <td><a href="Topic_10.html#wt0lvhfl">Wu_T_2009</a></td>
    </tr>
    <tr>
      <th id="q26f8pv4";>6</th>
      <td>Sperschneider_2010</td>
      <td>DotKnot: pseudoknot prediction using the probability dot plot under a refined energy model</td>
      <td>Sperschneider, Jana; Datta, Amitava</td>
      <td>2010</td>
      <td>2010-01-31</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853144" target="_blank">PMC2853144</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20123730.0" target="_blank">20123730.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkq021" target="_blank">10.1093/nar/gkq021</a></td>
      <td>q26f8pv4</td>
      <td>0.726748</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_10.html#gjild96e">Song_2005</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_10.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="p1i7bx94";>7</th>
      <td>Zhang_2013</td>
      <td>Enhancement of accuracy and efficiency for RNA secondary structure prediction by sequence segmentation and MapReduce</td>
      <td>Zhang, Boyu; Yehdego, Daniel T; Johnson, Kyle L; Leung, Ming-Ying; Taufer, Michela</td>
      <td>2013</td>
      <td>2013-11-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3952952" target="_blank">PMC3952952</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24564983.0" target="_blank">24564983.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6807-13-s1-s3" target="_blank">10.1186/1472-6807-13-s1-s3</a></td>
      <td>p1i7bx94</td>
      <td>0.721104</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_04.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="gytg4iku";>8</th>
      <td>Vila_2018</td>
      <td>(13)C Chemical Shifts in Proteins: A Rich Source of Encoded Structural Information</td>
      <td>Vila, Jorge A.; Arnautova, Yelena A.</td>
      <td>2018</td>
      <td>2018-07-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123919" target="_blank">PMC7123919</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-95843-9_20" target="_blank">10.1007/978-3-319-95843-9_20</a></td>
      <td>gytg4iku</td>
      <td>0.686326</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_04.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_10.html#jz2mrn4t">Vila_2014</a></td>
    </tr>
    <tr>
      <th id="wt0lvhfl";>9</th>
      <td>Wu_T_2009</td>
      <td>IRSS: a web-based tool for automatic layout and analysis of IRES secondary structure prediction and searching system in silico</td>
      <td>Wu, Tzong-Yuan; Hsieh, Chi-Chun; Hong, Jun-Jie; Chen, Chung-Yung; Tsai, Yuh-Show</td>
      <td>2009</td>
      <td>2009-05-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2698906" target="_blank">PMC2698906</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19473520.0" target="_blank">19473520.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-160" target="_blank">10.1186/1471-2105-10-160</a></td>
      <td>wt0lvhfl</td>
      <td>0.683653</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_10.html#7pdh7alh">Hong_2013</a></td>
    </tr>
    <tr>
      <th id="1lojd0xa";>10</th>
      <td>Gao_J_2010</td>
      <td>Inverse folding of RNA pseudoknot structures</td>
      <td>Gao, James ZM; Li, Linda YM; Reidys, Christian M</td>
      <td>2010</td>
      <td>2010-06-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2909241" target="_blank">PMC2909241</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20573197.0" target="_blank">20573197.0</a></td>
      <td><a href="https://doi.org/10.1186/1748-7188-5-27" target="_blank">10.1186/1748-7188-5-27</a></td>
      <td>1lojd0xa</td>
      <td>0.667477</td>
      <td><a href="Topic_10.html#9zm4per4">Dawson_2007</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="5igorndt";>11</th>
      <td>Janssen_2015</td>
      <td>The RNA shapes studio</td>
      <td>Janssen, Stefan; Giegerich, Robert</td>
      <td>2015</td>
      <td>2015-02-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4308662" target="_blank">PMC4308662</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25273103.0" target="_blank">25273103.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btu649" target="_blank">10.1093/bioinformatics/btu649</a></td>
      <td>5igorndt</td>
      <td>0.645316</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_02.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_10.html#1lojd0xa">Gao_J_2010</a>, <a href="Topic_10.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="jz2mrn4t";>12</th>
      <td>Vila_2014</td>
      <td>(13)C Chemical Shifts in Proteins: A Rich Source of Encoded Structural Information</td>
      <td>Vila, Jorge A.; Arnautova, Yelena A.</td>
      <td>2014</td>
      <td>2014-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121069" target="_blank">PMC7121069</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-642-28554-7_19" target="_blank">10.1007/978-3-642-28554-7_19</a></td>
      <td>jz2mrn4t</td>
      <td>0.629744</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_04.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_10.html#gytg4iku">Vila_2018</a></td>
    </tr>
    <tr>
      <th id="wtvfow2f";>13</th>
      <td>Seemann_2008</td>
      <td>Unifying evolutionary and thermodynamic information for RNA folding of multiple alignments</td>
      <td>Seemann, Stefan E.; Gorodkin, Jan; Backofen, Rolf</td>
      <td>2008</td>
      <td>2008-10-04</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2582601" target="_blank">PMC2582601</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18836192.0" target="_blank">18836192.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkn544" target="_blank">10.1093/nar/gkn544</a></td>
      <td>wtvfow2f</td>
      <td>0.626505</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="jms7hrmp";>14</th>
      <td>Liu_C_2005</td>
      <td>Profiling and Searching for RNA Pseudoknot Structures in Genomes</td>
      <td>Liu, Chunmei; Song, Yinglei; Malmberg, Russell L.; Cai, Liming</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120494" target="_blank">PMC7120494</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11567752_2" target="_blank">10.1007/11567752_2</a></td>
      <td>jms7hrmp</td>
      <td>0.618832</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_10.html#egbkl2v2">Liu_C_2005</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#egbkl2v2">Liu_C_2005</a></td>
    </tr>
    <tr>
      <th id="u14tj0so";>15</th>
      <td>Jabbari_2014</td>
      <td>A fast and robust iterative algorithm for prediction of RNA pseudoknotted secondary structures</td>
      <td>Jabbari, Hosna; Condon, Anne</td>
      <td>2014</td>
      <td>2014-05-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4064103" target="_blank">PMC4064103</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24884954.0" target="_blank">24884954.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-147" target="_blank">10.1186/1471-2105-15-147</a></td>
      <td>u14tj0so</td>
      <td>0.604465</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="9zm4per4";>16</th>
      <td>Dawson_2007</td>
      <td>Prediction of RNA Pseudoknots Using Heuristic Modeling with Mapping and Sequential Folding</td>
      <td>Dawson, Wayne K.; Fujiwara, Kazuya; Kawai, Gota</td>
      <td>2007</td>
      <td>2007-09-19</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1975678" target="_blank">PMC1975678</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17878940.0" target="_blank">17878940.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0000905" target="_blank">10.1371/journal.pone.0000905</a></td>
      <td>9zm4per4</td>
      <td>0.583955</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="iomwwsud";>17</th>
      <td>Sadreyev_2003</td>
      <td>Profile–profile comparisons by COMPASS predict intricate homologies between protein families</td>
      <td>Sadreyev, Ruslan I.; Baker, David; Grishin, Nick V.</td>
      <td>2003</td>
      <td>2003-10-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2366929" target="_blank">PMC2366929</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/14500884.0" target="_blank">14500884.0</a></td>
      <td></td>
      <td>iomwwsud</td>
      <td>0.571006</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#s7kzp795">Zemla_2011</a></td>
    </tr>
    <tr>
      <th id="1t433pfu";>18</th>
      <td>Vila_2010</td>
      <td>Sequential nearest-neighbor effects on computed (13)C(α) chemical shifts</td>
      <td>Vila, Jorge A.; Serrano, Pedro; Wüthrich, Kurt; Scheraga, Harold A.</td>
      <td>2010</td>
      <td>2010-07-20</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2970923" target="_blank">PMC2970923</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20644980.0" target="_blank">20644980.0</a></td>
      <td><a href="https://doi.org/10.1007/s10858-010-9435-7" target="_blank">10.1007/s10858-010-9435-7</a></td>
      <td>1t433pfu</td>
      <td>0.562064</td>
      <td><a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="iuglkdcp";>19</th>
      <td>Sperschneider_2012</td>
      <td>Predicting pseudoknotted structures across two RNA sequences</td>
      <td>Sperschneider, Jana; Datta, Amitava; Wise, Michael J.</td>
      <td>2012</td>
      <td>2012-12-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3516145" target="_blank">PMC3516145</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23044552.0" target="_blank">23044552.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bts575" target="_blank">10.1093/bioinformatics/bts575</a></td>
      <td>iuglkdcp</td>
      <td>0.555985</td>
      <td><a href="Topic_02.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="azpg5yrh";>20</th>
      <td>Mead_2019</td>
      <td>Visualization of protein sequence space with force-directed graphs, and their application to the choice of target-template pairs for homology modelling</td>
      <td>Mead, Dylan J.T.; Lunagomez, Simón; Gatherer, Derek</td>
      <td>2019</td>
      <td>2019-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110651" target="_blank">PMC7110651</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31377535.0" target="_blank">31377535.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jmgm.2019.07.014" target="_blank">10.1016/j.jmgm.2019.07.014</a></td>
      <td>azpg5yrh</td>
      <td>0.552688</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_10.html#s7kzp795">Zemla_2011</a>, <a href="Topic_04.html#u1imic5l">Li_C_2018</a>, <a href="Topic_04.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="e9app2cv";>21</th>
      <td>Lu_C_2005</td>
      <td>A memory-efficient algorithm for multiple sequence alignment with constraints</td>
      <td>Lu, Chin Lung; Huang, Yen Pin</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109922" target="_blank">PMC7109922</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15374876.0" target="_blank">15374876.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bth468" target="_blank">10.1093/bioinformatics/bth468</a></td>
      <td>e9app2cv</td>
      <td>0.547978</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#gjild96e">Song_2005</a>, <a href="Topic_10.html#cjcvm3x7">Chung_2007</a>, <a href="Topic_01.html#zuh78iul">Pessia_2015</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="p8puu1jk";>22</th>
      <td>Li_M_2015</td>
      <td>Predicting RNA Secondary Structures: One-grammar-fits-all Solution</td>
      <td>Li, Menglu; Cheng, Micheal; Ye, Yongtao; Hon, Wk; Ting, Hf; Lam, Tw; Tang, Cy; Wong, Thomas; Yiu, Sm</td>
      <td>2015</td>
      <td>2015-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121278" target="_blank">PMC7121278</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-19048-8_18" target="_blank">10.1007/978-3-319-19048-8_18</a></td>
      <td>p8puu1jk</td>
      <td>0.546844</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#1lojd0xa">Gao_J_2010</a></td>
    </tr>
    <tr>
      <th id="9z8z03rp";>23</th>
      <td>Kai_Z_2019</td>
      <td>An efficient simulated annealing algorithm for the RNA secondary structure prediction with Pseudoknots</td>
      <td>Kai, Zhang; Yuting, Wang; Yulin, Lv; Jun, Liu; Juanjuan, He</td>
      <td>2019</td>
      <td>2019-12-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6933665" target="_blank">PMC6933665</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31881969.0" target="_blank">31881969.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-019-6300-2" target="_blank">10.1186/s12864-019-6300-2</a></td>
      <td>9z8z03rp</td>
      <td>0.545533</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#jxgd1yc4">Kai_Z_2018</a>, <a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="6f1vqkqm";>24</th>
      <td>Zhao_2013</td>
      <td>SSW Library: An SIMD Smith-Waterman C/C++ Library for Use in Genomic Applications</td>
      <td>Zhao, Mengyao; Lee, Wan-Ping; Garrison, Erik P.; Marth, Gabor T.</td>
      <td>2013</td>
      <td>2013-12-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852983" target="_blank">PMC3852983</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24324759.0" target="_blank">24324759.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0082138" target="_blank">10.1371/journal.pone.0082138</a></td>
      <td>6f1vqkqm</td>
      <td>0.540064</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_10.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_10.html#zgv4kd6j">Reeder_2007</a></td>
    </tr>
    <tr>
      <th id="r9diwa8y";>25</th>
      <td>Bendaña_2008</td>
      <td>Colorstock, SScolor, Ratón: RNA alignment visualization tools</td>
      <td>Bendaña, Yuri R.; Holmes, Ian H.</td>
      <td>2008</td>
      <td>2008-02-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109877" target="_blank">PMC7109877</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18218657.0" target="_blank">18218657.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btm635" target="_blank">10.1093/bioinformatics/btm635</a></td>
      <td>r9diwa8y</td>
      <td>0.533783</td>
      <td><a href="Topic_02.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_15.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_10.html#zgv4kd6j">Reeder_2007</a></td>
    </tr>
    <tr>
      <th id="k23xlzj0";>26</th>
      <td>Delli Ponti_2017</td>
      <td>A high-throughput approach to profile RNA structure</td>
      <td>Delli Ponti, Riccardo; Marti, Stefanie; Armaos, Alexandros; Tartaglia, Gian Gaetano</td>
      <td>2017</td>
      <td>2017-03-17</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389523" target="_blank">PMC5389523</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27899588.0" target="_blank">27899588.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkw1094" target="_blank">10.1093/nar/gkw1094</a></td>
      <td>k23xlzj0</td>
      <td>0.533647</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_02.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="zgv4kd6j";>27</th>
      <td>Reeder_2007</td>
      <td>pknotsRG: RNA pseudoknot folding including near-optimal structures and sliding windows</td>
      <td>Reeder, Jens; Steffen, Peter; Giegerich, Robert</td>
      <td>2007</td>
      <td>2007-05-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933184" target="_blank">PMC1933184</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17478505.0" target="_blank">17478505.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm258" target="_blank">10.1093/nar/gkm258</a></td>
      <td>zgv4kd6j</td>
      <td>0.518743</td>
      <td><a href="Topic_02.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_02.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_02.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="egbkl2v2";>28</th>
      <td>Liu_C_2005</td>
      <td>Profiling and Searching for RNA Pseudoknot Structures in Genomes</td>
      <td>Liu, Chunmei; Song, Yinglei; Malmberg, Russell L.; Cai, Liming</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122704" target="_blank">PMC7122704</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11428848_123" target="_blank">10.1007/11428848_123</a></td>
      <td>egbkl2v2</td>
      <td>0.515520</td>
      <td><a href="Topic_10.html#jms7hrmp">Liu_C_2005</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#jms7hrmp">Liu_C_2005</a></td>
    </tr>
    <tr>
      <th id="jxgd1yc4";>29</th>
      <td>Kai_Z_2018</td>
      <td>A Novel Efficient Simulated Annealing Algorithm for the RNA Secondary Structure Predicting with Pseudoknots</td>
      <td>Kai, Zhang; Yulin, Lv</td>
      <td>2018</td>
      <td>2018-07-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121944" target="_blank">PMC7121944</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-95933-7_44" target="_blank">10.1007/978-3-319-95933-7_44</a></td>
      <td>jxgd1yc4</td>
      <td>0.504126</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="qgit5qqo";>30</th>
      <td>Reeder_2006</td>
      <td>Beyond Mfold: Recent advances in RNA bioinformatics</td>
      <td>Reeder, Jens; Höchsmann, Matthias; Rehmsmeier, Marc; Voss, Björn; Giegerich, Robert</td>
      <td>2006</td>
      <td>2006-06-25</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125698" target="_blank">PMC7125698</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16530285.0" target="_blank">16530285.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbiotec.2006.01.034" target="_blank">10.1016/j.jbiotec.2006.01.034</a></td>
      <td>qgit5qqo</td>
      <td>0.486719</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="gjild96e";>31</th>
      <td>Song_2005</td>
      <td>Efficient Parameterized Algorithm for Biopolymer Structure-Sequence Alignment</td>
      <td>Song, Yinglei; Liu, Chunmei; Huang, Xiuzhen; Malmberg, Russell L.; Xu, Ying; Cai, Liming</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121179" target="_blank">PMC7121179</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11557067_31" target="_blank">10.1007/11557067_31</a></td>
      <td>gjild96e</td>
      <td>0.484580</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="pq1vaknj";>32</th>
      <td>Heal_2018</td>
      <td>Applying graph theory to protein structures: an Atlas of coiled coils</td>
      <td>Heal, Jack W; Bartlett, Gail J; Wood, Christopher W; Thomson, Andrew R; Woolfson, Derek N</td>
      <td>2018</td>
      <td>2018-10-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6157074" target="_blank">PMC6157074</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29722888.0" target="_blank">29722888.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bty347" target="_blank">10.1093/bioinformatics/bty347</a></td>
      <td>pq1vaknj</td>
      <td>0.437459</td>
      <td><a href="Topic_02.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_10.html#gjild96e">Song_2005</a>, <a href="Topic_13.html#g4k6pc82">Munteanu_2008</a>, <a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a></td>
    </tr>
    <tr>
      <th id="cjcvm3x7";>33</th>
      <td>Chung_2007</td>
      <td>RE-MuSiC: a tool for multiple sequence alignment with regular expression constraints</td>
      <td>Chung, Yun-Sheng; Lee, Wei-Hsun; Tang, Chuan Yi; Lu, Chin Lung</td>
      <td>2007</td>
      <td>2007-05-08</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933182" target="_blank">PMC1933182</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17488842.0" target="_blank">17488842.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm275" target="_blank">10.1093/nar/gkm275</a></td>
      <td>cjcvm3x7</td>
      <td>0.425949</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_10.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_10.html#gjild96e">Song_2005</a>, <a href="Topic_10.html#s7kzp795">Zemla_2011</a>, <a href="Topic_05.html#9zfct1yx">Altschul_1991</a></td>
    </tr>
    <tr>
      <th id="s7kzp795";>34</th>
      <td>Zemla_2011</td>
      <td>StralSV: assessment of sequence variability within similar 3D structures and application to polio RNA-dependent RNA polymerase</td>
      <td>Zemla, Adam T; Lang, Dorothy M; Kostova, Tanya; Andino, Raul; Ecale Zhou, Carol L</td>
      <td>2011</td>
      <td>2011-06-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3121648" target="_blank">PMC3121648</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21635786.0" target="_blank">21635786.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-226" target="_blank">10.1186/1471-2105-12-226</a></td>
      <td>s7kzp795</td>
      <td>0.364098</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="9qv11m4f";>35</th>
      <td>Heo_L_2020</td>
      <td>Modeling of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Proteins by Machine Learning and Physics-Based Refinement</td>
      <td>Lim Heo; Michael Feig</td>
      <td>2020</td>
      <td>2020-03-28</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.25.008904" target="_blank">10.1101/2020.03.25.008904</a></td>
      <td>9qv11m4f</td>
      <td>0.335310</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="zj1fvery";>36</th>
      <td>Roy_R_2007</td>
      <td>3.36 Application of Multivalent Mannosylated Dendrimers in Glycobiology</td>
      <td>Roy, R.; Touaibia, M.</td>
      <td>2007</td>
      <td>2007-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7152399" target="_blank">PMC7152399</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-044451967-2/00112-4" target="_blank">10.1016/b978-044451967-2/00112-4</a></td>
      <td>zj1fvery</td>
      <td>0.306352</td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>
</body>
</html>
