
<!DOCTYPE html>
<html>
<meta charset="UTF-8">
<head>
  <title>Topic 04 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</title>

<style>
table {
  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #0099FF;
  color: white;
}
</style>

</head>
<body>
<h2>Topic 04 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</h2>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cite ad</th>
      <th>title</th>
      <th>authors</th>
      <th>publish year</th>
      <th>publish time</th>
      <th>dataset</th>
      <th>abstract mentions covid</th>
      <th>pmcid</th>
      <th>pubmed id</th>
      <th>doi</th>
      <th>cord uid</th>
      <th>topic weight</th>
      <th>Similarity scispacy</th>
      <th>Similarity specter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="w49x7kca";>1</th>
      <td>Biao_2016</td>
      <td>Accurate prediction of human essential genes using only nucleotide composition and association information</td>
      <td>Feng-Biao Biao; Chuan Dong; Hong-Li Hua; Shuo Liu; Hao Luo; Hong-Wan Zhang; Yan-Ting Jin; Kai-Yue Zhang</td>
      <td>2016</td>
      <td>2016-10-28</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/084129" target="_blank">10.1101/084129</a></td>
      <td>w49x7kca</td>
      <td>0.936510</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#vj9tv3or">Guo_F_2017</a>, <a href="Topic_04.html#qyd6qyjf">Xiao_2017</a></td>
    </tr>
    <tr>
      <th id="vj9tv3or";>2</th>
      <td>Guo_F_2017</td>
      <td>Accurate prediction of human essential genes using only nucleotide composition and association information</td>
      <td>Guo, Feng-Biao; Dong, Chuan; Hua, Hong-Li; Liu, Shuo; Luo, Hao; Zhang, Hong-Wan; Jin, Yan-Ting; Zhang, Kai-Yue</td>
      <td>2017</td>
      <td>2017-06-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110051" target="_blank">PMC7110051</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28158612.0" target="_blank">28158612.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btx055" target="_blank">10.1093/bioinformatics/btx055</a></td>
      <td>vj9tv3or</td>
      <td>0.921651</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#w49x7kca">Biao_2016</a>, <a href="Topic_04.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_04.html#98oo0bkl">Nakai_1992</a>, <a href="Topic_04.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="rx0os4ke";>3</th>
      <td>Wu_G_2007</td>
      <td>Prediction of mutations engineered by randomness in H5N1 hemagglutinins of influenza A virus</td>
      <td>Wu, G.; Yan, S.</td>
      <td>2007</td>
      <td>2007-11-02</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088403" target="_blank">PMC7088403</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17973072.0" target="_blank">17973072.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-007-0602-4" target="_blank">10.1007/s00726-007-0602-4</a></td>
      <td>rx0os4ke</td>
      <td>0.912639</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#7qc1lqyp">Cheng_2012</a></td>
      <td><a href="Topic_04.html#wocuomb1">Wu_G_2007</a></td>
    </tr>
    <tr>
      <th id="rktwtqtq";>4</th>
      <td>Cai_C_2006</td>
      <td>Prediction of Transmembrane Proteins from Their Primary Sequence by Support Vector Machine Approach</td>
      <td>Cai, C. Z.; Yuan, Q. F.; Xiao, H. G.; Liu, X. H.; Han, L. Y.; Chen, Y. Z.</td>
      <td>2006</td>
      <td>2006-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121931" target="_blank">PMC7121931</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11816102_56" target="_blank">10.1007/11816102_56</a></td>
      <td>rktwtqtq</td>
      <td>0.858167</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#88l9bq4i">Han_L_2005</a></td>
    </tr>
    <tr>
      <th id="7vfo98ho";>5</th>
      <td>Zhao_2012</td>
      <td>Using Support Vector Machine and Evolutionary Profiles to Predict Antifreeze Protein Sequences</td>
      <td>Zhao, Xiaowei; Ma, Zhiqiang; Yin, Minghao</td>
      <td>2012</td>
      <td>2012-02-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3292016" target="_blank">PMC3292016</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22408447.0" target="_blank">22408447.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms13022196" target="_blank">10.3390/ijms13022196</a></td>
      <td>7vfo98ho</td>
      <td>0.836657</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_11.html#v4xue7ha">Xu_Y_2015</a></td>
    </tr>
    <tr>
      <th id="8c75hp7r";>6</th>
      <td>Muthukrishnan_2014</td>
      <td>Support vector machine (SVM) based multiclass prediction with basic statistical analysis of plasminogen activators</td>
      <td>Muthukrishnan, Selvaraj; Puri, Munish; Lefevre, Christophe</td>
      <td>2014</td>
      <td>2014-01-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3924408" target="_blank">PMC3924408</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24468032.0" target="_blank">24468032.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-7-63" target="_blank">10.1186/1756-0500-7-63</a></td>
      <td>8c75hp7r</td>
      <td>0.835704</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#rktwtqtq">Cai_C_2006</a></td>
    </tr>
    <tr>
      <th id="kt6wqvam";>7</th>
      <td>Xiao_2013</td>
      <td>iGPCR-Drug: A Web Server for Predicting Interaction between GPCRs and Drugs in Cellular Networking</td>
      <td>Xiao, Xuan; Min, Jian-Liang; Wang, Pu; Chou, Kuo-Chen</td>
      <td>2013</td>
      <td>2013-08-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3754978" target="_blank">PMC3754978</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24015221.0" target="_blank">24015221.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0072234" target="_blank">10.1371/journal.pone.0072234</a></td>
      <td>kt6wqvam</td>
      <td>0.814814</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_04.html#qdld7hdc">Fan_Y_2014</a>, <a href="Topic_04.html#jpj1czhg">Min_J_2013</a></td>
    </tr>
    <tr>
      <th id="7floluv1";>8</th>
      <td>Yan_S_2009</td>
      <td>Mutation patterns in human α-galactosidase A</td>
      <td>Yan, Shaomin; Wu, Guang</td>
      <td>2009</td>
      <td>2009-05-26</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088632" target="_blank">PMC7088632</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19468850.0" target="_blank">19468850.0</a></td>
      <td><a href="https://doi.org/10.1007/s11030-009-9158-4" target="_blank">10.1007/s11030-009-9158-4</a></td>
      <td>7floluv1</td>
      <td>0.792340</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="wocuomb1";>9</th>
      <td>Wu_G_2007</td>
      <td>Prediction of mutations engineered by randomness in H5N1 neuraminidases from influenza A virus</td>
      <td>Wu, G.; Yan, S.</td>
      <td>2007</td>
      <td>2007-08-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088166" target="_blank">PMC7088166</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17721674.0" target="_blank">17721674.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-007-0579-z" target="_blank">10.1007/s00726-007-0579-z</a></td>
      <td>wocuomb1</td>
      <td>0.786540</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#7qc1lqyp">Cheng_2012</a></td>
      <td><a href="Topic_04.html#rx0os4ke">Wu_G_2007</a></td>
    </tr>
    <tr>
      <th id="6oca1mrm";>10</th>
      <td>Shen_2018</td>
      <td>RPiRLS: Quantitative Predictions of RNA Interacting with Any Protein of Known Sequence</td>
      <td>Shen, Wen-Jun; Cui, Wenjuan; Chen, Danze; Zhang, Jieming; Xu, Jianzhen</td>
      <td>2018</td>
      <td>2018-02-28</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6017498" target="_blank">PMC6017498</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29495575.0" target="_blank">29495575.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules23030540" target="_blank">10.3390/molecules23030540</a></td>
      <td>6oca1mrm</td>
      <td>0.770435</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_04.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="ni6iyzdn";>11</th>
      <td>He_Z_2010</td>
      <td>Predicting Drug-Target Interaction Networks Based on Functional Groups and Biological Features</td>
      <td>He, Zhisong; Zhang, Jian; Shi, Xiao-He; Hu, Le-Le; Kong, Xiangyin; Cai, Yu-Dong; Chou, Kuo-Chen</td>
      <td>2010</td>
      <td>2010-03-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836373" target="_blank">PMC2836373</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20300175.0" target="_blank">20300175.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0009603" target="_blank">10.1371/journal.pone.0009603</a></td>
      <td>ni6iyzdn</td>
      <td>0.761389</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_07.html#fewy8y4a">Wang_2018</a>, <a href="Topic_07.html#bs17igjs">Zhou_2010</a></td>
    </tr>
    <tr>
      <th id="3fwd4guk";>12</th>
      <td>Xu_Y_2013</td>
      <td>iSNO-AAPair: incorporating amino acid pairwise coupling into PseAAC for predicting cysteine S-nitrosylation sites in proteins</td>
      <td>Xu, Yan; Shao, Xiao-Jian; Wu, Ling-Yun; Deng, Nai-Yang; Chou, Kuo-Chen</td>
      <td>2013</td>
      <td>2013-10-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3792191" target="_blank">PMC3792191</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24109555.0" target="_blank">24109555.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.171" target="_blank">10.7717/peerj.171</a></td>
      <td>3fwd4guk</td>
      <td>0.759284</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_04.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="85fqseal";>13</th>
      <td>Lee_B_2009</td>
      <td>Identification of protein functions using a machine-learning approach based on sequence-derived properties</td>
      <td>Lee, Bum Ju; Shin, Moon Sun; Oh, Young Joon; Oh, Hae Seok; Ryu, Keun Ho</td>
      <td>2009</td>
      <td>2009-08-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2731080" target="_blank">PMC2731080</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19664241.0" target="_blank">19664241.0</a></td>
      <td><a href="https://doi.org/10.1186/1477-5956-7-27" target="_blank">10.1186/1477-5956-7-27</a></td>
      <td>85fqseal</td>
      <td>0.754843</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_04.html#88l9bq4i">Han_L_2005</a></td>
    </tr>
    <tr>
      <th id="8k7d1riv";>14</th>
      <td>Muppirala_2011</td>
      <td>Predicting RNA-Protein Interactions Using Only Sequence Information</td>
      <td>Muppirala, Usha K; Honavar, Vasant G; Dobbs, Drena</td>
      <td>2011</td>
      <td>2011-12-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322362" target="_blank">PMC3322362</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22192482.0" target="_blank">22192482.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-489" target="_blank">10.1186/1471-2105-12-489</a></td>
      <td>8k7d1riv</td>
      <td>0.745388</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_04.html#6oca1mrm">Shen_2018</a></td>
    </tr>
    <tr>
      <th id="ch4kere3";>15</th>
      <td>Yan_C_2019</td>
      <td>IILLS: predicting virus-receptor interactions based on similarity and semi-supervised learning</td>
      <td>Yan, Cheng; Duan, Guihua; Wu, Fang-Xiang; Wang, Jianxin</td>
      <td>2019</td>
      <td>2019-12-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6933616" target="_blank">PMC6933616</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31881820.0" target="_blank">31881820.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-3278-3" target="_blank">10.1186/s12859-019-3278-3</a></td>
      <td>ch4kere3</td>
      <td>0.732932</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_08.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_08.html#55lkfivh">Cho_M_2019</a></td>
    </tr>
    <tr>
      <th id="88l9bq4i";>16</th>
      <td>Han_L_2005</td>
      <td>Prediction of functional class of novel viral proteins by a statistical learning method irrespective of sequence similarity</td>
      <td>Han, L.Y.; Cai, C.Z.; Ji, Z.L.; Chen, Y.Z.</td>
      <td>2005</td>
      <td>2005-01-05</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111859" target="_blank">PMC7111859</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15582660.0" target="_blank">15582660.0</a></td>
      <td><a href="https://doi.org/10.1016/j.virol.2004.10.020" target="_blank">10.1016/j.virol.2004.10.020</a></td>
      <td>88l9bq4i</td>
      <td>0.723577</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_04.html#85fqseal">Lee_B_2009</a></td>
    </tr>
    <tr>
      <th id="qdld7hdc";>17</th>
      <td>Fan_Y_2014</td>
      <td>iNR-Drug: Predicting the Interaction of Drugs with Nuclear Receptors in Cellular Networking</td>
      <td>Fan, Yue-Nong; Xiao, Xuan; Min, Jian-Liang; Chou, Kuo-Chen</td>
      <td>2014</td>
      <td>2014-03-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3975431" target="_blank">PMC3975431</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24651462.0" target="_blank">24651462.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms15034915" target="_blank">10.3390/ijms15034915</a></td>
      <td>qdld7hdc</td>
      <td>0.698522</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#kt6wqvam">Xiao_2013</a></td>
    </tr>
    <tr>
      <th id="jpj1czhg";>18</th>
      <td>Min_J_2013</td>
      <td>iEzy-Drug: A Web Server for Identifying the Interaction between Enzymes and Drugs in Cellular Networking</td>
      <td>Min, Jian-Liang; Xiao, Xuan; Chou, Kuo-Chen</td>
      <td>2013</td>
      <td>2013-11-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3858977" target="_blank">PMC3858977</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24371828.0" target="_blank">24371828.0</a></td>
      <td><a href="https://doi.org/10.1155/2013/701317" target="_blank">10.1155/2013/701317</a></td>
      <td>jpj1czhg</td>
      <td>0.693429</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="bhtl73z9";>19</th>
      <td>Suresh_2015</td>
      <td>RPI-Pred: predicting ncRNA-protein interaction using sequence and structural information</td>
      <td>Suresh, V.; Liu, Liang; Adjeroh, Donald; Zhou, Xiaobo</td>
      <td>2015</td>
      <td>2015-02-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4330382" target="_blank">PMC4330382</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25609700.0" target="_blank">25609700.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkv020" target="_blank">10.1093/nar/gkv020</a></td>
      <td>bhtl73z9</td>
      <td>0.687048</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_04.html#6oca1mrm">Shen_2018</a></td>
    </tr>
    <tr>
      <th id="59yvbupx";>20</th>
      <td>Scott_2011</td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td>Scott, Michelle S; Troshin, Peter V; Barton, Geoffrey J</td>
      <td>2011</td>
      <td>2011-08-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3166288" target="_blank">PMC3166288</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21812952.0" target="_blank">21812952.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-317" target="_blank">10.1186/1471-2105-12-317</a></td>
      <td>59yvbupx</td>
      <td>0.675233</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_02.html#6qpsxmgi">Wang_2006</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="78vjzrpx";>21</th>
      <td>Song_2018</td>
      <td>iProt-Sub: a comprehensive package for accurately mapping and predicting protease-specific substrates and cleavage sites</td>
      <td>Song, Jiangning; Wang, Yanan; Li, Fuyi; Akutsu, Tatsuya; Rawlings, Neil D; Webb, Geoffrey I; Chou, Kuo-Chen</td>
      <td>2018</td>
      <td>2018-04-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6556904" target="_blank">PMC6556904</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29897410.0" target="_blank">29897410.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bby028" target="_blank">10.1093/bib/bby028</a></td>
      <td>78vjzrpx</td>
      <td>0.670094</td>
      <td><a href="Topic_01.html#my2l85bn">Cho_Y_2005</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="v1p4ut66";>22</th>
      <td>Lockwood_2011</td>
      <td>Neighborhood Properties Are Important Determinants of Temperature Sensitive Mutations</td>
      <td>Lockwood, Svetlana; Krishnamoorthy, Bala; Ye, Ping</td>
      <td>2011</td>
      <td>2011-12-02</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3229608" target="_blank">PMC3229608</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22164302.0" target="_blank">22164302.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0028507" target="_blank">10.1371/journal.pone.0028507</a></td>
      <td>v1p4ut66</td>
      <td>0.626946</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ldze0a22";>23</th>
      <td>Wang_2018</td>
      <td>Prediction of RNA-protein interactions using conjoint triad feature and chaos game representation</td>
      <td>Wang, Hongchu; Wu, Pengfei</td>
      <td>2018</td>
      <td>2018-08-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6984769" target="_blank">PMC6984769</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30117758.0" target="_blank">30117758.0</a></td>
      <td><a href="https://doi.org/10.1080/21655979.2018.1470721" target="_blank">10.1080/21655979.2018.1470721</a></td>
      <td>ldze0a22</td>
      <td>0.613570</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_04.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_04.html#6oca1mrm">Shen_2018</a>, <a href="Topic_04.html#qyd6qyjf">Xiao_2017</a></td>
    </tr>
    <tr>
      <th id="ob37jivi";>24</th>
      <td>Milanetti_2020</td>
      <td>In-Silico evidence for two receptors based strategy of SARS-CoV-2</td>
      <td>Edoardo Milanetti; Mattia Miotto; Lorenzo Di Rienzo; Michele Monti; Giorgio Gosti; Giancarlo Ruocco</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.24.006197" target="_blank">10.1101/2020.03.24.006197</a></td>
      <td>ob37jivi</td>
      <td>0.582781</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="7qriyc1a";>25</th>
      <td>Yang_2007</td>
      <td>Predicting Palmitoylation Sites Using a Regularised Bio-basis Function Neural Network</td>
      <td>Yang, Zheng Rong</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120480" target="_blank">PMC7120480</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72031-7_37" target="_blank">10.1007/978-3-540-72031-7_37</a></td>
      <td>7qriyc1a</td>
      <td>0.573232</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_07.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="ut9z9eo7";>26</th>
      <td>Mu_Z_2019</td>
      <td>DCGR: feature extractions from protein sequences based on CGR via remodeling multiple information</td>
      <td>Mu, Zengchao; Yu, Ting; Qi, Enfeng; Liu, Juntao; Li, Guojun</td>
      <td>2019</td>
      <td>2019-06-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6587251" target="_blank">PMC6587251</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31221087.0" target="_blank">31221087.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-2943-x" target="_blank">10.1186/s12859-019-2943-x</a></td>
      <td>ut9z9eo7</td>
      <td>0.565937</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#u1imic5l">Li_C_2018</a>, <a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#c3ewuebx">Jia_J_2016</a></td>
    </tr>
    <tr>
      <th id="fsh3gsvd";>27</th>
      <td>Xiao_2005</td>
      <td>Using cellular automata images and pseudo amino acid composition to predict protein subcellular location</td>
      <td>Xiao, X.; Shao, S.; Ding, Y.; Huang, Z.; Chou, K.-C.</td>
      <td>2005</td>
      <td>2005-07-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087770" target="_blank">PMC7087770</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16044193.0" target="_blank">16044193.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-005-0225-6" target="_blank">10.1007/s00726-005-0225-6</a></td>
      <td>fsh3gsvd</td>
      <td>0.563120</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_04.html#98oo0bkl">Nakai_1992</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_04.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="u1imic5l";>28</th>
      <td>Li_C_2018</td>
      <td>Protein Sequence Comparison and DNA-binding Protein Identification with Generalized PseAAC and Graphical Representation</td>
      <td>Li, Chun; Zhao, Jialing; Wang, Changzhong; Yao, Yuhua</td>
      <td>2018</td>
      <td>2018-02-23</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5930480" target="_blank">PMC5930480</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29380690.0" target="_blank">29380690.0</a></td>
      <td><a href="https://doi.org/10.2174/1386207321666180130100838" target="_blank">10.2174/1386207321666180130100838</a></td>
      <td>u1imic5l</td>
      <td>0.554277</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="c3ewuebx";>29</th>
      <td>Jia_J_2016</td>
      <td>iPPBS-Opt: A Sequence-Based Ensemble Classifier for Identifying Protein-Protein Binding Sites by Optimizing Imbalanced Training Datasets</td>
      <td>Jia, Jianhua; Liu, Zi; Xiao, Xuan; Liu, Bingxiang; Chou, Kuo-Chen</td>
      <td>2016</td>
      <td>2016-01-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6274413" target="_blank">PMC6274413</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26797600.0" target="_blank">26797600.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules21010095" target="_blank">10.3390/molecules21010095</a></td>
      <td>c3ewuebx</td>
      <td>0.502995</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_07.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="66qoykl1";>30</th>
      <td>Shulman-Peleg_2004</td>
      <td>Recognition of Functional Sites in Protein Structures</td>
      <td>Shulman-Peleg, Alexandra; Nussinov, Ruth; Wolfson, Haim J.</td>
      <td>2004</td>
      <td>2004-06-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126412" target="_blank">PMC7126412</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15147845.0" target="_blank">15147845.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jmb.2004.04.012" target="_blank">10.1016/j.jmb.2004.04.012</a></td>
      <td>66qoykl1</td>
      <td>0.495980</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="0ec2xunt";>31</th>
      <td>Plewczyński_2008</td>
      <td>The interactome: Predicting the protein-protein interactions in cells</td>
      <td>Plewczyński, Dariusz; Ginalski, Krzysztof</td>
      <td>2008</td>
      <td>2008-10-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6275871" target="_blank">PMC6275871</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18839074.0" target="_blank">18839074.0</a></td>
      <td><a href="https://doi.org/10.2478/s11658-008-0024-7" target="_blank">10.2478/s11658-008-0024-7</a></td>
      <td>0ec2xunt</td>
      <td>0.492799</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a></td>
    </tr>
    <tr>
      <th id="54ioyzbc";>32</th>
      <td>Xiao_2005</td>
      <td>Using cellular automata to generate image representation for biological sequences</td>
      <td>Xiao, X.; Shao, S.; Ding, Y.; Huang, Z.; Chen, X.; Chou, K.-C.</td>
      <td>2005</td>
      <td>2005-02-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088382" target="_blank">PMC7088382</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15700108.0" target="_blank">15700108.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-004-0154-9" target="_blank">10.1007/s00726-004-0154-9</a></td>
      <td>54ioyzbc</td>
      <td>0.485631</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_04.html#fsh3gsvd">Xiao_2005</a>, <a href="Topic_10.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="64zqe4cz";>33</th>
      <td>Al-Ssulami_2020</td>
      <td>CodSeqGen: A tool for generating synonymous coding sequences with desired GC-contents</td>
      <td>Al-Ssulami, Abdulrakeeb M.; Azmi, Aqil M.; Hussain, Muhammad</td>
      <td>2020</td>
      <td>2020-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127556" target="_blank">PMC7127556</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30738198.0" target="_blank">30738198.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ygeno.2019.02.002" target="_blank">10.1016/j.ygeno.2019.02.002</a></td>
      <td>64zqe4cz</td>
      <td>0.473835</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="yjrs7c38";>34</th>
      <td>De_Ferrari_2014</td>
      <td>From sequence to enzyme mechanism using multi-label machine learning</td>
      <td>De Ferrari, Luna; Mitchell, John BO</td>
      <td>2014</td>
      <td>2014-05-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4229970" target="_blank">PMC4229970</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24885296.0" target="_blank">24885296.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-150" target="_blank">10.1186/1471-2105-15-150</a></td>
      <td>yjrs7c38</td>
      <td>0.470770</td>
      <td><a href="Topic_02.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="qyd6qyjf";>35</th>
      <td>Xiao_2017</td>
      <td>Prediction of lncRNA-protein interactions using HeteSim scores based on heterogeneous networks</td>
      <td>Xiao, Yun; Zhang, Jingpu; Deng, Lei</td>
      <td>2017</td>
      <td>2017-06-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5473862" target="_blank">PMC5473862</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28623317.0" target="_blank">28623317.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-017-03986-1" target="_blank">10.1038/s41598-017-03986-1</a></td>
      <td>qyd6qyjf</td>
      <td>0.458325</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_04.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="my2l85bn";>36</th>
      <td>Cho_Y_2005</td>
      <td>Rule Generation Using NN and GA for SARS-CoV Cleavage Site Prediction</td>
      <td>Cho, Yeon-Jin; Kim, Hyeoncheol</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122303" target="_blank">PMC7122303</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11553939_111" target="_blank">10.1007/11553939_111</a></td>
      <td>my2l85bn</td>
      <td>0.458313</td>
      <td><a href="Topic_01.html#gc9hevy5">Cho_Y_2005</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="2scgsmsc";>37</th>
      <td>Chou_2011</td>
      <td>Some remarks on protein attribute prediction and pseudo amino acid composition</td>
      <td>Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125570" target="_blank">PMC7125570</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21168420.0" target="_blank">21168420.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2010.12.024" target="_blank">10.1016/j.jtbi.2010.12.024</a></td>
      <td>2scgsmsc</td>
      <td>0.434131</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_04.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_04.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_04.html#fsh3gsvd">Xiao_2005</a>, <a href="Topic_04.html#yjrs7c38">De_Ferrari_2014</a></td>
    </tr>
    <tr>
      <th id="gc9hevy5";>38</th>
      <td>Cho_Y_2005</td>
      <td>Cleavage Site Analysis Using Rule Extraction from Neural Networks</td>
      <td>Cho, Yeun-Jin; Kim, Hyeoncheol</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114972" target="_blank">PMC7114972</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11539087_132" target="_blank">10.1007/11539087_132</a></td>
      <td>gc9hevy5</td>
      <td>0.430876</td>
      <td><a href="Topic_01.html#my2l85bn">Cho_Y_2005</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_01.html#my2l85bn">Cho_Y_2005</a></td>
    </tr>
    <tr>
      <th id="m56agj4z";>39</th>
      <td>Reddy_2007</td>
      <td>Positional clustering improves computational binding site detection and identifies novel cis-regulatory sites in mammalian GABA(A) receptor subunit genes</td>
      <td>Reddy, Timothy E.; Shakhnovich, Boris E.; Roberts, Daniel S.; Russek, Shelley J.; DeLisi, Charles</td>
      <td>2007</td>
      <td>2007-01-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1807961" target="_blank">PMC1807961</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17204484.0" target="_blank">17204484.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl1062" target="_blank">10.1093/nar/gkl1062</a></td>
      <td>m56agj4z</td>
      <td>0.420210</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="0dxrai3j";>40</th>
      <td>Alguwaizani_2018</td>
      <td>Predicting Interactions between Virus and Host Proteins Using Repeat Patterns and Composition of Amino Acids</td>
      <td>Alguwaizani, Saud; Park, Byungkyu; Zhou, Xiang; Huang, De-Shuang; Han, Kyungsook</td>
      <td>2018</td>
      <td>2018-05-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5966669" target="_blank">PMC5966669</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29854357.0" target="_blank">29854357.0</a></td>
      <td><a href="https://doi.org/10.1155/2018/1391265" target="_blank">10.1155/2018/1391265</a></td>
      <td>0dxrai3j</td>
      <td>0.408352</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_08.html#4uvezfqm">Zhou_2018</a></td>
    </tr>
    <tr>
      <th id="6l688krc";>41</th>
      <td>Hung_2006</td>
      <td>Alignment using genetic programming with causal trees for identification of protein functions</td>
      <td>Hung, Chun-Min; Huang, Yueh-Min; Chang, Ming-Shi</td>
      <td>2006</td>
      <td>2006-09-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117053" target="_blank">PMC7117053</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.na.2005.09.048" target="_blank">10.1016/j.na.2005.09.048</a></td>
      <td>6l688krc</td>
      <td>0.401178</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="ol2du7ph";>42</th>
      <td>Kliger_2008</td>
      <td>Predicting proteolytic sites in extracellular proteins: only halfway there</td>
      <td>Kliger, Yossef; Gofer, Eyal; Wool, Assaf; Toporik, Amir; Apatoff, Avihay; Olshansky, Moshe</td>
      <td>2008</td>
      <td>2008-04-15</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109841" target="_blank">PMC7109841</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18321887.0" target="_blank">18321887.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btn084" target="_blank">10.1093/bioinformatics/btn084</a></td>
      <td>ol2du7ph</td>
      <td>0.399385</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="t54zftal";>43</th>
      <td>Kumar_2017</td>
      <td>Prediction of endoplasmic reticulum resident proteins using fragmented amino acid composition and support vector machine</td>
      <td>Kumar, Ravindra; Kumari, Bandana; Kumar, Manish</td>
      <td>2017</td>
      <td>2017-09-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5588793" target="_blank">PMC5588793</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28890846.0" target="_blank">28890846.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.3561" target="_blank">10.7717/peerj.3561</a></td>
      <td>t54zftal</td>
      <td>0.392767</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="g1ob16b9";>44</th>
      <td>Xie_X_2012</td>
      <td>Protein sequence analysis based on hydropathy profile of amino acids</td>
      <td>Xie, Xiao-li; Zheng, Li-fei; Yu, Ying; Liang, Li-ping; Guo, Man-cai; Song, John; Yuan, Zhi-fa</td>
      <td>2012</td>
      <td>2012-01-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274743" target="_blank">PMC3274743</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22302429.0" target="_blank">22302429.0</a></td>
      <td><a href="https://doi.org/10.1631/jzus.b1100052" target="_blank">10.1631/jzus.b1100052</a></td>
      <td>g1ob16b9</td>
      <td>0.388857</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="7bh8222u";>45</th>
      <td>Zhang_2007</td>
      <td>Estimating residue evolutionary conservation by introducing von Neumann entropy and a novel gap-treating approach</td>
      <td>Zhang, S.-W.; Zhang, Y.-L.; Pan, Q.; Cheng, Y.-M.; Chou, K.-C.</td>
      <td>2007</td>
      <td>2007-08-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088136" target="_blank">PMC7088136</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17710364.0" target="_blank">17710364.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-007-0586-0" target="_blank">10.1007/s00726-007-0586-0</a></td>
      <td>7bh8222u</td>
      <td>0.381550</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_01.html#zuh78iul">Pessia_2015</a>, <a href="Topic_05.html#9zfct1yx">Altschul_1991</a></td>
    </tr>
    <tr>
      <th id="0mn1gbll";>46</th>
      <td>Hu_L_2011</td>
      <td>Predicting Biological Functions of Compounds Based on Chemical-Chemical Interactions</td>
      <td>Hu, Le-Le; Chen, Chen; Huang, Tao; Cai, Yu-Dong; Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-12-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3248422" target="_blank">PMC3248422</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22220213.0" target="_blank">22220213.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0029491" target="_blank">10.1371/journal.pone.0029491</a></td>
      <td>0mn1gbll</td>
      <td>0.375421</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#fewy8y4a">Wang_2018</a></td>
    </tr>
    <tr>
      <th id="98oo0bkl";>47</th>
      <td>Nakai_1992</td>
      <td>A knowledge base for predicting protein localization sites in eukaryotic cells</td>
      <td>Nakai, Kenta; Kanehisa, Minoru</td>
      <td>1992</td>
      <td>1992-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134799" target="_blank">PMC7134799</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/1478671.0" target="_blank">1478671.0</a></td>
      <td><a href="https://doi.org/10.1016/s0888-7543(05)80111-9" target="_blank">10.1016/s0888-7543(05)80111-9</a></td>
      <td>98oo0bkl</td>
      <td>0.364921</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_04.html#59yvbupx">Scott_2011</a></td>
    </tr>
    <tr>
      <th id="hjo78o1q";>48</th>
      <td>Shen_2006</td>
      <td>Probabilistic analysis of the frequencies of amino acid pairs within characterized protein sequences</td>
      <td>Shen, Shiyi; Kai, Bo; Ruan, Jishou; Torin Huzil, J.; Carpenter, Eric; Tuszynski, Jack A.</td>
      <td>2006</td>
      <td>2006-10-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127678" target="_blank">PMC7127678</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.physa.2006.03.004" target="_blank">10.1016/j.physa.2006.03.004</a></td>
      <td>hjo78o1q</td>
      <td>0.358200</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ppzxvykb";>49</th>
      <td>Dinkel_2011</td>
      <td>ELM—the database of eukaryotic linear motifs</td>
      <td>Dinkel, Holger; Michael, Sushama; Weatheritt, Robert J.; Davey, Norman E.; Van Roey, Kim; Altenberg, Brigitte; Toedt, Grischa; Uyar, Bora; Seiler, Markus; Budd, Aidan; Jödicke, Lisa; Dammert, Marcel A.; Schroeter, Christian; Hammer, Maria; Schmidt, Tobias; Jehl, Peter; McGuigan, Caroline; Dymecka, Magdalena; Chica, Claudia; Luck, Katja; Via, Allegra; Chatr-aryamontri, Andrew; Haslam, Niall; Grebnev, Gleb; Edwards, Richard J.; Steinmetz, Michel O.; Meiselbach, Heike; Diella, Francesca; Gibson, Toby J.</td>
      <td>2011</td>
      <td>2011-11-21</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245074" target="_blank">PMC3245074</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22110040.0" target="_blank">22110040.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkr1064" target="_blank">10.1093/nar/gkr1064</a></td>
      <td>ppzxvykb</td>
      <td>0.338062</td>
      <td><a href="Topic_02.html#6qpsxmgi">Wang_2006</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="2dbpm0nt";>50</th>
      <td>Choo_2009</td>
      <td>A comprehensive assessment of N-terminal signal peptides prediction methods</td>
      <td>Choo, Khar Heng; Tan, Tin Wee; Ranganathan, Shoba</td>
      <td>2009</td>
      <td>2009-12-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2788353" target="_blank">PMC2788353</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19958512.0" target="_blank">19958512.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-s15-s2" target="_blank">10.1186/1471-2105-10-s15-s2</a></td>
      <td>2dbpm0nt</td>
      <td>0.337763</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a></td>
    </tr>
    <tr>
      <th id="4uvezfqm";>51</th>
      <td>Zhou_2018</td>
      <td>A generalized approach to predicting protein-protein interactions between virus and host</td>
      <td>Zhou, Xiang; Park, Byungkyu; Choi, Daesik; Han, Kyungsook</td>
      <td>2018</td>
      <td>2018-08-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6101077" target="_blank">PMC6101077</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30367586.0" target="_blank">30367586.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-018-4924-2" target="_blank">10.1186/s12864-018-4924-2</a></td>
      <td>4uvezfqm</td>
      <td>0.330133</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="hkvll3xh";>52</th>
      <td>Yang_2009</td>
      <td>Peptide Bioinformatics- Peptide Classification Using Peptide Machines</td>
      <td>Yang, Zheng Rong</td>
      <td>2009</td>
      <td>2009-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122642" target="_blank">PMC7122642</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19065810.0" target="_blank">19065810.0</a></td>
      <td><a href="https://doi.org/10.1007/978-1-60327-101-1_9" target="_blank">10.1007/978-1-60327-101-1_9</a></td>
      <td>hkvll3xh</td>
      <td>0.318403</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="s7kzp795";>53</th>
      <td>Zemla_2011</td>
      <td>StralSV: assessment of sequence variability within similar 3D structures and application to polio RNA-dependent RNA polymerase</td>
      <td>Zemla, Adam T; Lang, Dorothy M; Kostova, Tanya; Andino, Raul; Ecale Zhou, Carol L</td>
      <td>2011</td>
      <td>2011-06-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3121648" target="_blank">PMC3121648</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21635786.0" target="_blank">21635786.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-226" target="_blank">10.1186/1471-2105-12-226</a></td>
      <td>s7kzp795</td>
      <td>0.317472</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4y5mnok8";>54</th>
      <td>Ehrt_2018</td>
      <td>A benchmark driven guide to binding site comparison: An exhaustive evaluation using tailor-made data sets (ProSPECCTs)</td>
      <td>Ehrt, Christiane; Brinkjost, Tobias; Koch, Oliver</td>
      <td>2018</td>
      <td>2018-11-08</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6224041" target="_blank">PMC6224041</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30408032.0" target="_blank">30408032.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pcbi.1006483" target="_blank">10.1371/journal.pcbi.1006483</a></td>
      <td>4y5mnok8</td>
      <td>0.310112</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#d66rz6dc">Webb_2014</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_07.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="n10ia3j3";>55</th>
      <td>Liu_H_2008</td>
      <td>A new approach to the prediction of transmembrane structures</td>
      <td>Liu, HongDe; Wang, Rui; Lu, XiaoQuan; Chen, Jing; Liu, Xiuhui; Ding, Lan</td>
      <td>2008</td>
      <td>2008-05-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088861" target="_blank">PMC7088861</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11434-008-0055-5" target="_blank">10.1007/s11434-008-0055-5</a></td>
      <td>n10ia3j3</td>
      <td>0.301214</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="er6lz09f";>56</th>
      <td>Pfab_2020</td>
      <td>DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes</td>
      <td>Jonas Pfab; Dong Si</td>
      <td>2020</td>
      <td>2020-02-13</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.12.946772" target="_blank">10.1101/2020.02.12.946772</a></td>
      <td>er6lz09f</td>
      <td>0.300949</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#oohx14kl">Zhong_2020</a></td>
    </tr>
  </tbody>
</table>
</body>
</html>
